application/forms/Command/Object/DeleteDowntimeForm.php

```diff
@@ -25,11 +27,29 @@ protected function assembleElements()
 
     protected function assembleSubmitButton()
     {
+        $isDisabled = false;
+        foreach ($this->getObjects() as $downtime) {
+            if ($downtime->scheduled_by) {
+                $downtimeName = $downtime->name;
+                $scheduledBy = $downtime->scheduled_by;
+                $isDisabled = true;
+                break;
+            }
+        }
+
         $this->addElement(
             'submitButton',
             'btn_submit',
             [
-                'class' => ['cancel-button', 'spinner'],
+                'class' => ['cancel-button', 'spinner', $isDisabled ? 'disabled' : null],
+                'disabled' => $isDisabled ?: null,
+                'title' => $isDisabled
+                    ? sprintf(
+                        t('Cannot remove downtime %s as it is owned by scheduled downtime object %s.'),
```

Comment:
Don't use such a technical message. (This is directly from the api response isn't it?) Just tell the user that it's based on a configured scheduled downtime and hence cannot be removed at runtime.